src/pages/RoomMembersPage.tsx

```diff
@@ -195,6 +186,17 @@ function RoomMembersPage({report, policies}: RoomMembersPageProps) {
         return activeParticipants.length >= CONST.SHOULD_SHOW_MEMBERS_SEARCH_INPUT_BREAKPOINT;
     }, [participants, personalDetails, isOffline, report]);
 
+    useEffect(() => {
+        if (!isFocusedScreen || !shouldShowTextInput) {
+            return;
+        }
+        setSearchValue(userSearchPhrase ?? '');
+    }, [isFocusedScreen, shouldShowTextInput, userSearchPhrase]);
+
+    useEffect(() => {
+        UserSearchPhraseActions.updateUserSearchPhrase(searchValue);
+    }, [searchValue]);
+
```

Comment:
hm just realizing isn't this basically the same as what already exists immediately below it? 

https://github.com/Expensify/App/blob/3f2c9b511959d343a31cde01a6d5c7e011bb61b3/src/pages/RoomMembersPage.tsx#L189-L210

Reply:
Ooh looks veeeeery similar!

Reply:
yeah good point, maybe we should combine?